data/language/en-GB.txt

```diff
@@ -3695,6 +3695,10 @@ STR_6589    :Show window buttons on the left
 STR_6590    :Show the window buttons (e.g. to close the window) on the left of the title bar instead of on the right.
 STR_6591    :Staff member is currently fixing a ride and can’t be fired.
 STR_6592    :Staff member is currently inspecting a ride and can’t be fired.
+STR_6593    :LSM Launched Roller Coaster
+STR_6594    :A low to the ground roller coaster propelled by Liniar Synchronous Motors.
```

Comment:
How about "Roller coaster trains are accelerated by linear synchronous motors, speeding through low-to-the-ground twists and inversions"

Also I'll throw my vote in for "Multi-Launch" over "LSM Launched", probably would be clearer to the average player

Reply:
I would drop the phrase "low to the ground", they don't have to be. I was going to say it should be rephrased as "to speed through twisting inversions" but that is almost word for word what the LIM launched coaster says (not that it necessarily has to be different I guess?)

Reply:
My idea of the LSM Launched Roller Coaster's description would be this:
**_Powerful linear synchronous motors boost roller coaster trains out of the station to tackle high-speed twists and inversions_**